### ISSUE TITLE
Openrouter not working

### ISSUE THREAD

**Issue by PSG2017** [logancyang/obsidian-copilot#529]
Hi

No matter what I do I can't get Openrouter to respond. I can chat on the Openrouter site but not from Copilot. I have API loaded and it indicates what model I'm using when I select OPENROUTER.AI . WHen I send my query appears in the chat, then the bot icon appears momentarily and then disappears again. Then nothing happens.

OpenAI working just fine.

**Comment by PSG2017** [comment]
Now got it working thanks

**Comment by logancyang** [comment]
Were you able to identify the issue?

**Comment by PSG2017** [comment]
Yes thanks, the issue was that when I asked Openrouter for a key it did not display the key for copying, so I was copying the truncated version on the keys page, thinking that that was the full key. On the fourth attempt Openrouter finally displayed the full key for copying. Sorry, novice error, I have never before asked Openrouter for a key. BTW this Copilot plugin is a remarkable achievement - it opens up so many possibilities.
